backend/db/utils.py

```diff
@@ -277,3 +415,30 @@ def list_tables(con: Connection, schema: str = SCHEMA) -> List[str]:
         WHERE schemaname in ('{schema}') ORDER BY 1;"""
     result = run_sql(con, query)
     return [x[0] for x in result]
+
+
+def get_ddl_statements(schema: str = SCHEMA, modules: List[str] = None, table_suffix='') -> List[str]:
```

Comment:
FYI: The logic that now controls the DDL. This is used both by `initalize()` and the DB refresh.

It also allows the addition of a `table_suffix`, so that I can create a copy of a table w/ a suffix, e.g. `_new`, and then rename the table and drop the old one, so that there is no downtime in data availability while the refresh is going on.